pkgs/applications/version-management/git-and-tools/git/default.nix

```diff
@@ -70,7 +72,9 @@ stdenv.mkDerivation {
   # so that `SPARSE_FLAGS' corresponds to the current architecture...
   #doCheck = true;
 
-  installFlags = "NO_INSTALL_HARDLINKS=1";
+  installFlags = "NO_INSTALL_HARDLINKS=1"
+    + (if withpcre2 then " USE_LIBPCRE2=1" else "");
```

Comment:
Is `libpcre2` already part of the dependency closure? Then we can just enable it. An alternative would be to add it only to `gitFull`. I have no opinion, whether it should be also available in the `git` derivation.